aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -355,4 +355,8 @@ void cleanupMetadataAndUnlock(CommitStatus commitStatus, String metadataLocation
       }
     }
   }
+
+  public Map<String, String> tableCatalogProperties() {
```

Comment:
looks like this is only for testing purpose? can we use spy for that instead of exposing it?